test/ooni/http_invalid_request_line_test.cpp

```diff
@@ -76,3 +76,16 @@ TEST_CASE("Make sure that it can pass options to the other levels") {
     } while (!done);
     for (auto &s : *logs) std::cout << s << "\n";
 }
+
+TEST_CASE("Make sure that the test can deal with an invalid backend") {
+    Var<std::list<std::string>> logs(new std::list<std::string>);
+    bool done = false;
+    ooni::HttpInvalidRequestLineTest()
+        .set_backend("nexacenter.org")
+        .on_log([=](const char *s) { logs->push_back(s); })
+        .run([&done]() { done = true; });
```

Comment:
/me puzzled: Shouldn't `run()` be blocking and only terminate at the end of the test?